core/src/main/java/it/pagopa/selfcare/pagopa/backoffice/core/GecServiceImpl.java

```diff
@@ -41,5 +39,11 @@ public String createPSPBundle(String idpsp, BundleCreate bundle){
         String response = gecConnector.createPSPBundle(idpsp, bundle);
         return response;
     }
+    @Override
+    @Cacheable(value = "GecPaymentTypes")
```

Comment:
the key of this cache must be dynamic because the result depends on the inputs (`limit and page`). 

We have two options:
1. remove `cacheable`
2. add a dynamic cache key